starter-code/css/main.css

```diff
@@ -1,8 +1,168 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    margin:0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Varela Round', sans-serif;
+    /*line-height: 3.5em;*/
+    color:#1A1A1A;
+}
+
+.nav-bar {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color:#FFF;
+    width: 100%;
+
+}
+
+.nav-bar ul{
+    width: 40%;
+    margin-right: 20px;
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-bar ul li{
+    padding: 2px;
+    font-size: 1em;
+}
+
+.nav-bar div img {
+    width: 125px;
+}
+
+.conteiner {
+    background-image:url("./../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: transparent;
+    height: 650px;
+    padding-top: 50px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+}
+
+h1 {
+    font-weight: lighter;
+    font-size: 3em;
+    text-align: center;
+    color:#FFF;
+    letter-spacing: 4px;
+}
+
+h2 {
+    font-weight: lighter;
+    font-size: 1.25em;
+    text-align: center;
+    color:#FFF;
+    line-height: 1%;
+    letter-spacing: 0.6px;
+}
+
+#section1 {
+    width: 100%;
+    text-align: center;
+}
+
+#section1 h3 {
+    border-bottom: 3px solid #00B172;
+    display: inline-block;
+}
+
+.icons {
+    display: flex;
+    justify-content: space-around;
+}
+
+.service-box {
+    width: 200px;
+    height: 200px;
+    text-align: center;
+    box-sizing: border-box;
+    line-height: 1px;
+}
+
+#section1 > h4 {
+    font-weight: bold;
+    font-size: 1em;
+    color: #00B172;
+
+}
+
+.service-box img {
+    width: 100px;
+}
+
+.service-box p {
+    font-weight: lighter;
+    font-size: 0.8em;
+}
+
+.green-box {
+    width: 100%;
+    height: 650px;
+    background-image:url("../images/spotify-icon-white.png");
+    background-position: center;
+    background-size: 100px;
+    background-repeat: no-repeat;
+    background-color: #00B172;
+    position: relative;
```

Comment:
You positioned your .green-box relatively, this causes your nav-bar, to appear behind the green-box element when you scroll down.
You can either remove this property because you don't position any element inside your green box absolutely, or you could increase the z-index on the nav-bar to be higher than the green box so it would appear infront of it.